hicommonwealth/commonwealth
# Deprecate bulkMembers endpoint

## Link to Issue
Closes: #3987

## Description of Changes
- Removes UI usages of the `bulkMembers` endpoint in favor of `searchProfiles` and `listRoles` in the Edit Collaborators modal and Manage Community page, respectively.
- Removes `bulkMembers` endpoint.

## Test Plan
- Go to the Manage Community page, add and delete a member as admin/mod– confirm that delete works correctly
- Go to a Thread View page, open the Edit Collaborators modal– confirm that you can search for a user, add them and delete them

## Deployment Plan
N/A

## Other Considerations
N/A
